src/css/components/reviews.css

```diff
@@ -0,0 +1,102 @@
+.reviews__list {
+  margin-top: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.reviews__list li:nth-last-child(-n+2) {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .reviews__list {
+    flex-wrap: wrap;
+    margin-top: 64px;
```

Comment:
do we really need to use margin-top? Because section-head may have margin-bottom